build_macos_dev.sh

```
@@ -119,6 +168,7 @@ function patchWebRTC() {
     patch -b -p0 -d $WORK_DIR < $PATCHES_DIR/task_factory.patch
     patch -b -p0 -d $WORK_DIR < $PATCHES_DIR/metal_header.patch
     patch -b -p0 -d $WORK_DIR < $PATCHES_DIR/echo_cancellation.patch
+    patch -b -p0 -d $WORK_DIR < $PATCHES_DIR/nsgl_deprecation.patch
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)